akka-actor/src/main/scala/akka/io/SimpleDnsManager.scala

```diff
@@ -42,11 +42,13 @@ class SimpleDnsManager(val ext: DnsExt) extends Actor with RequiresMessageQueue[
 
     case m: dns.DnsProtocol.Resolve ⇒
       if (inetDnsEnabled) {
-        log.error(
+        log.info(
```

Comment:
Why isn't this a user error?

Reply:
error is fatal stuff, this is "meh plz dont" but will not crash anything. Also old versions of the "rp" stuff do this, so our own tools now cause error logging. We'll fix it by fixing it and lagom but in general I dont think this is error, since we handle it gracefully anyway

Reply:
Sorry I thought about this again in more depth and you are right... when this one is used but async comes in we can fail...

Reply:
Will amend the message though, there is no `akka.io.dns.DnsProtocol.resolve`